index.html

```diff
@@ -140,8 +140,27 @@ <h3>LET'S CONNECT</h3>
       <li class="language-item"><img src="./Bullets/html.png" alt="html-logo"><p>HTML</p></li>
       <li class="language-item"><img src="./Bullets/css.png" alt="css-logo"><p>CSS</p></li>
       <li>Frameworks<hr></li>
-      <li>Skills</li>
+      <li>Skills<hr></li>
     </ul>
-  </section>
+  </section><br>
+  <form class="form" action="https://formspree.io/f/xdobanld" method="POST">
```

Comment:
Nice work applying the formspree functionality 😎 

_The form has been sent successfully_ ✔

![image](https://user-images.githubusercontent.com/15807343/149033251-822bec8e-d6d8-4bb7-be12-a4cfc70c5c6b.png)